### DISCUSSION TITLE
INCAR setting error in atomate workflows

### DISCUSSION THREAD

**Discussion by adaile** [materialsproject/pymatgen#2976] [Q&A]
https://github.com/materialsproject/pymatgen/blob/9c7d2bb6f8f0b21474ed39b0bb4c9dce772ac240/pymatgen/io/vasp/sets.py#L441
When I calculate a Si test by using a atomate workflow, the INCAR include MAGMOM label, and the second step calculation stop.  How should I deal with this problem.

**Comment by janosh** [comment]
I think you'll have to provide more information to get useful answers. I'm converting this to a discussion since this doesn't sound like a code issue.
